tools/ci_build/github/linux/docker/build_scripts/finalize.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
All newly introduced files need license header.

Reply:
They are not our code. I put a LICENCE file in the folder.

Reply:
I see. Ok, can we add a README file in this folder with a line saying all files in this file have been copied from https://github.com/pypa/manylinux. It's not obvious for newcomers who might maintain this later.

Reply:
Added.